downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -91,69 +100,137 @@ ALLOW connection from controller(s) to receptor port
 .Execution Nodes
 [options="header"]
 |===
-|Port |Protocol |Service |Installer Inventory Variable |Required for
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
 |22
 |TCP
 |SSH
-|ansible_port
+|Bi-directional
+|`ansible_port`
 |Installation
 
 |27199
 |TCP
 |Receptor
-|receptor_listener_port
-a|Mesh - Nodes directly peered to controllers. No hop nodes involved. 27199 is bidirectional for the execution nodes
+|Bi-directional
+|`receptor_listener_port`
+a|Mesh - Nodes directly peered to controllers. No hop nodes involved. 27199 is bi-directional for the execution nodes
 
 ALLOW connections from controller(s) to receptor port (for non-hop connected nodes)
 
 ALLOW connections from hop node(s) to receptor port (if relayed through hop nodes)
 |===
 
+.Control Nodes
+[options="header"]
+|===
+|Port |Protocol |Service |Direction |Installer Inventory Variable |Required for
+|22
+|TCP
+|SSH
+|Bi-directional
+|`ansible_port`
+|Installation
+
+|27199
+|TCP
+|Receptor
+|Bi-directional
+|`receptor_listener_port`
+a|Mesh - Nodes directly peered to controllers. No hop nodes involved. 27199 is bi-diretional for execution nodes
+
+ENABLE connections from controller(s) to receptor port for non-hop connected nodes
+
+ENABLE connections from hop node(s) to receptor port if relayed through hop nodes
```

Comment:
Same comments for these two sentences. `s/receptor port/Receptor node` and change "non-hop" to direct. It's either a direct connection or a relayed connection. "non-hop" sounds weird.